### PR TITLE
fix/ gallery layouts defaults

### PR DESCRIPTION
**Pull request by talevy17**

### CODE REVIEW

File: packages/plugin-gallery/web/src/gallery-viewer.jsx [hunk 1]
```diff
@@ -93,9 +91,32 @@ class GalleryViewer extends React.Component {
...
+  shouldConsiderThumbnailSize = () => {
+    const { galleryLayout, galleryThumbnailsAlignment } = this.state.styleParams;
+    return (
+      galleryLayout === GALLERY_LAYOUTS.THUMBNAIL &&
+      ['top', 'bottom'].includes(galleryThumbnailsAlignment)
+    );
+  };
+
+  getGalleryHeight = width => {
+    let height;
```
Comment (shaulgo) [inline packages/plugin-gallery/web/src/gallery-viewer.jsx:103]:
```suggestion
    let height = 305;
```

File: packages/plugin-gallery/web/src/gallery-viewer.jsx [hunk 2]
```diff
@@ -93,9 +91,32 @@ class GalleryViewer extends React.Component {
...
+    if (width) {
+      height = Math.floor((width * 3) / 4);
+      if (this.shouldConsiderThumbnailSize()) {
+        height = this.props.isMobile ? Math.floor((width * 2) / 3) + 92.5 : Math.max(height, 585);
```
Comment (shaulgo) [inline packages/plugin-gallery/web/src/gallery-viewer.jsx:107]: is mobile needed?

File: packages/plugin-gallery/web/src/gallery-viewer.jsx [hunk 3]
```diff
@@ -167,9 +188,17 @@ class GalleryViewer extends React.Component {
     if (!this.props.isMobile) {
       return { ...styleParams, allowHover: true };
     }
+    let mobileParams = styleParams;
+    if (isHorizontalLayout(styleParams)) {
+      mobileParams = { ...mobileParams, arrowsSize: 20, imageMargin: 0 };
```
Comment (shaulgo) [inline packages/plugin-gallery/web/src/gallery-viewer.jsx:193]:
```suggestion
      mobileParams.arrowsSize = 20;
      mobileParams.imageMargin = 0;
```

File: packages/plugin-gallery/web/src/gallery-viewer.jsx [hunk 4]
```diff
@@ -167,9 +188,17 @@ class GalleryViewer extends React.Component {
...
+      mobileParams =
+        styleParams.galleryLayout === GALLERY_LAYOUTS.THUMBNAIL
+          ? { ...mobileParams, thumbnailSize: 90 }
+          : mobileParams;
```
Comment (shaulgo) [inline packages/plugin-gallery/web/src/gallery-viewer.jsx:197]:
```suggestion
	if (styleParams.galleryLayout === GALLERY_LAYOUTS.THUMBNAIL) {
	mobileParams.thumbnailSize = 90;
	}
```

**Review by shaulgo — CHANGES_REQUESTED** [review]

File: packages/plugin-gallery/web/src/gallery-viewer.jsx (hunk 2 above)
Comment (talevy17) [reply packages/plugin-gallery/web/src/gallery-viewer.jsx]: yes, the 2:3 ration (+95) is relevant for mobile only, on Desktop we want to have enough height to show the image properly and not cut it (that's why the Math.max is there)

**Review by shaulgo — APPROVED** [review]

**Pull request merged**